chain/chain/src/store.rs

```diff
@@ -2205,6 +2250,87 @@ impl<'a> ChainStoreUpdate<'a> {
         Ok(())
     }
 
+    // Clearing block data of `block_hash`, for the purpose of undo-block tool
```

Comment:
I find `for the purpose` comments not helpful. I'd much prefer to describe what the function does. And maybe somebody in the future decides that this function is useful in more usecases.